components/shared_data_model/requirements/requirements-dev.txt

```diff
@@ -209,7 +209,6 @@ lxml==4.9.1 \
     --hash=sha256:4780677767dd52b99f0af1f123bc2c22873d30b474aa0e2fc3fe5e02217687c7 \
     --hash=sha256:4878e667ebabe9b65e785ac8da4d48886fe81193a84bbe49f12acff8f7a383a4 \
     --hash=sha256:487c8e61d7acc50b8be82bda8c8d21d20e133c3cbf41bd8ad7eb1aaeb3f07c97 \
-    --hash=sha256:49a866923e69bc7da45a0565636243707c22752fc38f6b9d5c8428a86121022c \
```

Comment:
This is odd

Reply:
This is exactly the hash that pip-audit was complaining about:

```
pip_audit._service.interface.ServiceError: Mismatched hash for lxml (4.9.1): listed 49a866923e69bc7da45a0565636243707c22752fc38f6b9d5c8428a86121022c of type sha256 could not be found in PyPI releases
```

Maybe a dropped variant of lxml 4.9.1?

Reply:
I've checked PyPI but of course this hash can't be found there. Google has two hits, both lock files with this hash in it. Not much more we can do to research this I guess.